README.md

```diff
@@ -3,55 +3,76 @@
 ## Install
 
 - This assumes an environment is already set up for npm packages - if not, please use npm init following the steps at [https://docs.npmjs.com/getting-started/using-a-package.json](https://docs.npmjs.com/getting-started/using-a-package.json).
-- run `npm install @patternfly/patternfly-next --save`
+- run `npm install @patternfly/patternfly-next`
 
-When you install PatternFly Next, the package includes:
+Any of the files below are meant for use in consuming PatternFly's styles. The recommended consumption approach will vary from project to project.
 
-- a single file for the entire compiled library: `node_modules/@patternfly/patternfly-next/patternfly.css`
-- individual files with each component compiled separately: `node_modules/@patternfly/patternfly-next/<ComponentName>/styles.css`
-- a single file for the entire library's source (SASS): `node_modules/@patternfly/patternfly-next/patternfly.scss`
-- individual files for each component's source (SASS): `node_modules/@patternfly/patternfly-next/<ComponentName>/styles.scss`
+- a single file for the complete PatternFly styles:
 
-Any of the files above are meant for use in consuming the library. The recommended
-consumption approach will vary from project to project.
+  ```
+  node_modules/@patternfly/patternfly-next/patternfly.css
+  ```
+
+- individual stylesheets for each of PatternFly's blueprints (e.g. components, layout):
+
+  ```
+  node_modules/@patternfly/patternfly-next/<BlueprintType>/<BlueprintName>/<blueprint-name>.css
+  ```
+
+- the base source file for PatternFly's styles ([SASS](https://sass-lang.com/)):
+
+  ```
+  node_modules/@patternfly/patternfly-next/patternfly.scss
+  ```
+
+- individual source files for each PatternFly blueprint (SASS):
+
+  ```
+  node_modules/@patternfly/patternfly-next/<BlueprintType>/<BlueprintName>/<blueprint-name>.scss
+  ```
 
 ## Development
 
 **PatternFly Next Development requires Node v8.0.0 or greater**
 
-To setup the PatternFly Next development environment:
+To set up the PatternFly Next development environment:
 
 - clone the project
 - run `npm install` from the project root
-- run `npm run cli:setup` (only needed if doing development)
 - run `npm run dev`
 - open your browser to `http://localhost:8000`
 
-After working on your contribution, check for [accessibility violations](#testing-for-accessibility).
+After working on your contribution follow the guidelines in [Testing for Accessibility](#testing-for-accessibility), below.
+
+### Create new blueprints
```

Comment:
Please do not use the term blueprint out here - it's an internal term that has no meaning to the library.  The area below looks good generally, but I'd like to see the word blueprint replaced with Component/Demo/Layout/Utility.

Reply:
To confirm, change to "Create new components, layouts, utilities, and demos"? Makes sense